### PR TITLE
Add (very) simple Toshiba TEC (printer) support

### PR DESCRIPTION
**Pull request by rockyluke**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

### CODE REVIEW

File: includes/definitions/toshiba.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+os: toshiba
```
Comment (laf) [inline includes/definitions/toshiba.yaml]: I think we should change this to toshiba-printer, toshiba is too generic for a printer.

Comment (rockyluke) [inline includes/definitions/toshiba.yaml]: Good idea. It's fixed.

**Comment by rockyluke** [comment]
I've switched from toshiba to toshiba-printer.

**Comment by laf** [comment]
I've put the logo back as toshiba can remain generic for future devices.

**Comment by laf** [comment]
Actually toshiba-tec seemed more appropriate. I've updated and add unit tests.

**Pull request merged**